tools/recetox_aplcms/recetox_aplcms_correct_time.xml

```diff
@@ -44,7 +44,7 @@
     </inputs>
 
     <outputs>
-        <data label="${tool.name} on ${on_string}" name="output_file" format="parquet" />
+        <data label="${($features_table.element_identifier).rsplit('.',1)[0]}.parquet" name="output_file" format="parquet" />
```

Comment:
@zargham-ahmad Can you briefly explain what the label looked like before and after these changes? It is not clear from the code since some "Python magic" is done here. You can do it on an example.

Reply:
The element_identifier returns, for example, this "RCX_06_shortened.mzML", what I basically did is changed the ".mzML" with ".parquet" so the Name in the Dataset Information section will look like "RCX_06_shortened.parquet" for the recetox_aplcms tools we updated the labels for